### PR TITLE
ci(label): provide precise logs of check runs on skipping build

### LINKED ISSUE #3316: More logs in the label checks
### Problem to Solve

https://github.com/gear-tech/gear/actions/runs/6252495002/job/16975888383#step:3:19

the checks are not fully printed 

### Possible Solution

plus logs for the result of appending checks to the matched PRs

### Notes

ref https://github.com/gear-tech/gear/pull/3314

### LINKED ISSUE #3318: Generate & use `.binpath` in the `target` directory or reject it
### Problem to Solve

See the 6 point:
gear-foundation/cargo-program#103

### Possible Solution

This is only point related not only to `gear-wasm-builder` but also to `gtest`. So, we need to store `.binpath` somewhere in the `target` directory or reject it completely.

### Notes

_No response_

### PR DESCRIPTION
**Pull request by clearloop**
Resolves #3316

@gear-tech/dev

### CODE REVIEW

File: .github/actions/label/build.js [hunk 1]
```diff
@@ -30,29 +30,24 @@ const skip = async ({ core, github }) => {
...
-    // If there is already a build, skip this action without more conditions.
-    if (run.name === "build / linux" || run.name === "build / macos-x86")
+      run.name === "build / linux"
+      || run.name === "build / macos-x86"
+      || (run.name === "build" && run.conclusion !== "skipped")) {
       return true;
+    }
   }
 
-  return !skipped;
```
Comment (clearloop) [inline .github/actions/label/build.js:55]: #3474 not triggering build again, could be caused by we skipping this action by default, and only runs it on finding `build (status: skipped)`, which missed a situation that skipping `build` from github CI processes after this label CI.

instead, we skip build only on build has been processed without any other conditions, plus removing the confused variable `skipped`.

File: .github/actions/label/build.js [hunk 2]
```diff
@@ -30,29 +30,24 @@ const skip = async ({ core, github }) => {
...
-      (run.name === "build" && run.conclusion === "skipped")
-    )
-      skipped = true;
-
-    // If there is already a build, skip this action without more conditions.
-    if (run.name === "build / linux" || run.name === "build / macos-x86")
+      run.name === "build / linux"
+      || run.name === "build / macos-x86"
+      || (run.name === "build" && run.conclusion !== "skipped")) {
       return true;
```
Comment (breathx) [inline .github/actions/label/build.js:46]: Is it legit to return here just bool and below array with bool on first, further accepting only array for js?

Comment (breathx) [reply .github/actions/label/build.js:46]: @osipov-mit if it won't bother you, could you take a look pls?

**Review by breathx — APPROVED** [review]

File: .github/actions/label/build.js (hunk 2 above)
Comment (osipov-mit) [reply .github/actions/label/build.js:46]:
```suggestion
      return [true];
```

**Review by breathx — CHANGES_REQUESTED** [review]
Address comments above pls

File: .github/actions/label/build.js (hunk 2 above)
Comment (clearloop) [reply .github/actions/label/build.js:46]: it works in js ))

**Review by breathx — APPROVED** [review]

**Review by shamilsan — APPROVED** [review]

**Pull request merged**